README.md

```diff
@@ -154,64 +154,9 @@ cluster1-superuser@cqlsh> select * from system.peers;
 (2 rows)
 ```
 
-### (Optional) Loading the operator via Helm
-
-Helm may be used to install the operator. Consider installing it from our Helm Charts repo
-
-```console
-helm repo add datastax https://datastax.github.io/charts
-helm repo update
-
-# Helm 2
-helm install datastax/cass-operator
-
-# Helm 3
-helm install cass-operator datastax/cass-operator
-```
-
-_or via a local checkout_
-
-```console
-kubectl create namespace cass-operator-system
-helm install --namespace=cass-operator-system cass-operator ./charts/cass-operator-chart
-```
-
-The following Helm default values may be overridden:
-
-```yaml
-clusterWideInstall: false
-serviceAccountName: cass-operator
-clusterRoleName: cass-operator-cr
-clusterRoleBindingName: cass-operator-crb
-roleName: cass-operator
-roleBindingName: cass-operator
-webhookClusterRoleName: cass-operator-webhook
-webhookClusterRoleBindingName: cass-operator-webhook
-deploymentName: cass-operator
-deploymentReplicas: 1
-defaultImage: "datastax/cass-operator:1.6.0"
-imagePullPolicy: IfNotPresent
-imagePullSecret: ""
-```
-
-NOTE: roleName and roleBindingName will be used for a clusterRole and clusterRoleBinding if clusterWideInstall is set to true.
+### Installing cluster via Helm
 
-NOTE: Helm does not install a storage-class for the cassandra pods.
-
-If clusterWideInstall is set to true, then the operator will be able to administer `CassandraDatacenter`s in all namespaces of the kubernetes cluster.  A namespace must still be provided because some of the kubernetes resources for the operator require one.
-
-Example:
-
-```console
-kubectl create namespace cass-operator-system
-helm install --set clusterWideInstall=true --namespace=cass-operator-system cass-operator ./charts/cass-operator-chart
-```
-
-#### Using a custom Docker registry with the Helm Chart
-
-A custom Docker registry may be used as the source of the operator Docker image.  Before "helm install" is run, a Secret of type "docker-registry" should be created with the proper credentials.
-
-Then the "imagePullSecret" helm value may be set to the name of the ImagePullSecret to cause the custom Docker registry to be used.
+To install a cluster with optional integrated backup/restore and repair utilities, check the [k8ssandra/k8ssandra](https://github.com/k8ssandra/k8ssandra) helm charts project. 
```

Comment:
We should have a sentence referencing the full installation of K8ssandra, but still provide guidance on how to install just the `cass-operator` chart.

Not related to this PR, but interesting nonetheless, do you think there should be a Helm chart that simply templates out a `CassandraDatacenter`?

Reply:
I do not think that there should be a standalone chart to template out a CassandraDatacenter. By doing that you effectively constraint yourself to applying updates only through Helm. If you update the dc outside of Helm and then later run `helm upgrade` your changes are lost.